.github/workflows/conda-build.yml

```
@@ -39,4 +39,13 @@
         quantmsutilsc --help
-        quantmsutilsc --version
+        quantmsutilsc --version
+
+    - name: Download test data
+      run: |
+        wget https://ftp.pride.ebi.ac.uk/pub/databases/pride/resources/proteomes/quantms-ci-github/quantms-utils/TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.mzML
+        wget https://ftp.pride.ebi.ac.uk/pub/databases/pride/resources/proteomes/quantms-ci-github/quantms-utils/TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01_comet.idXML
+    - name: Test percolator ms2rescore
+      run: |
+        conda activate quantmsutils
+        quantmsutilsc ms2rescore --psm_file TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01_comet.idXML --spectrum_path TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.mzML --processes 2 --ms2pip_model HCD2021 --feature_generators 'ms2pip,deeplc' --id_decoy_pattern ^rev --test_fdr 0.05
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)